pkgs/tools/networking/hysteria/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib
+, fetchFromGitHub
+, buildGoModule
+}:
+buildGoModule rec {
+  pname = "hysteria";
+  version = "1.1.0";
+
+  src = fetchFromGitHub {
+    owner = "HyNetwork";
+    repo = "${pname}";
```

Comment:
I prefer the name and repo to be put here so overriding pname can be done without breakage. But yeah, I know there's bad examples around

Reply:
```nix
  pname = "hysteria";
  version = "1.1.0";

  src = fetchFromGitHub {
    owner = "HyNetwork";
    repo = "Hysteria";
    rev = "v${version}";
    sha256 = "sha256-V+umf7+qRANSNsMrU1Vij3ni6ayq/d41xSy3o+7sEHQ=";
  };
```
Is this the correct modification?

Reply:
Yeah, that looks good to me,

Reply:
@Mindavi Overriding `pnane` in this case is fine (and preferred, AFAIK, to avoid repetition) since it's not going off of the final value of `pname`, only the initial value.

If this was using overlay-style `mkDerivation`, you'd be correct. But that's not the case here.